src/main/cluster.ts

```diff
@@ -418,7 +419,7 @@ export class Cluster implements ClusterModel {
     } catch (error) {
       const ctx = this.getProxyKubeconfig().getContextObject(this.contextName)
       if (ctx.namespace) return [ctx.namespace]
-      return []
+      return this.accessibleNamespaces || [];
```

Comment:
If user has set namespaces manually we shouldn't even try to fetch full list from k8s api.

Reply:
Okay sounds good, I was wondering when that list should be used. Thanks for the clarification.